prometheus/registry.go

```diff
@@ -123,6 +133,16 @@ func MustRegister(m Collector) {
 	}
 }
 
+// MustRegisterAll works like MustRegister but takes a variadic list of
+// Collectors.
+func MustRegisterAll(m ...Collector) {
```

Comment:
How about making the normal `MustRegister` variadic instead? It doesn't break the current call syntax of `MustRegister` and avoids introducing yet another function.